internal/components/components.go

```diff
@@ -208,6 +209,7 @@ func Components() (component.Factories, error) {
 
 	exporters := []component.ExporterFactory{
 		alibabacloudlogserviceexporter.NewFactory(),
+		awscloudwatchlogsexporter.NewFactory(),
```

Comment:
I would recommend separating this out into its own PR to be merged once all of the work on the exporter is finished.  Eventually this will be moving to the collector-releases repo and will have to be separate.